app/client/components/delivery/address/deliveryAddressConfirmation.tsx

```diff
@@ -227,7 +227,10 @@ const AddressConfirmation = (props: ProductType) => {
 					<p
 						css={css`
 							${textSans.medium()};
-							margin-top: ${space[12]}px;
+							margin: ${space[12]}px 0
+								${showTopCallCentreNumbers
+									? `${space[5]}px`
+									: '0'};
```

Comment:
You could replace the logic for the bottom margin with a `<Stack>` wrapped around the paragraph and `<CallCentreEmailAndNumbers />` component? As it only applies spacing between adjacent elements it won't add any extra spacing if the call centre component isn't rendered. (Sorry, I've turned into a `<Stack>` cheerleader now 😀)

Reply:
👍  yep, good idea